plugins/gcp-projects/src/api/GcpClient.ts

```diff
@@ -17,6 +17,7 @@
 import { GcpApi } from './GcpApi';
 import { Operation, Project } from './types';
 import { OAuthApi } from '@backstage/core-plugin-api';
+import packageinfo from '../../version.json';
```

Comment:
```suggestion
import { version } from '../../package.json';
```

This is what i had in mind

Reply:
@Rugvip I tried this way, we have warnings linter errros
![Uploading image.png…]()

Reply:
Oh interesting, this is likely due to the new backend dev setup. Let's keep it as you had it originally, will have to look into this :+1:

Reply:
@Rugvip Push it commit with the originally way to bring back the package version.

Thank you!